docs/README.md

```diff
@@ -0,0 +1,16 @@
+---
```

Comment:
What's this file?

Reply:
the homepage ~ I thought maybe it could showcase our quick start script and the catalog of examples, along with other useful resources

pretty basic for now
![Screen Shot 2020-06-18 at 9 04 03 AM](https://user-images.githubusercontent.com/4283694/85023679-1024e500-b143-11ea-9ed1-ebb81f744c96.png)